src/components/Layout/Header/ActionCenter/components/SwapActionCard.tsx

```diff
@@ -108,14 +107,8 @@ export const SwapActionCard = ({
                 )}
               </HStack>
             </Stack>
-            {isCollapsable && swap?.txLink && (
-              <Icon
-                as={isOpen ? ChevronUpIcon : ChevronDownIcon}
-                ml='auto'
-                my='auto'
-                fontSize='xl'
-                color='text.subtle'
-              />
+            {!swap.isStreaming && swap.txLink && (
```

Comment:
suggestion: more of a product q but feels like we need a toggle/collapse arrow thingie for collapsed state, and external icon could be its own hitbox regardless of streaming swap (or perhaps, we want expanded states for non-streaming swaps too, and to move the Tx link to its own row in expanded details?)